extensions/wikia/Chat2/Chat.class.php

```diff
@@ -42,24 +55,21 @@ private static function permissionsService() {
 	 *
 	 * The key is then used by ChatAjax::getUserInfo() to load the info back from memcached.
 	 *
-	 * Here is the original description of this method, which is likely old and incorrect:
-	 * ---
-	 *
-	 * This function is meant to just echo the COOKIES which are available to the apache server.
-	 *
-	 * This helps to work around a limitation in the fact that javascript can't access the second-level-domain's
-	 * cookies even if they're authorized to be accessed by subdomains (eg: ".wikia.com" cookies are viewable by apache
-	 * on lyrics.wikia.com, but interestingly, isn't in document.cookie in the javascript on lyrics.wikia.com).
+	 * @return string
 	 */
-	public static function echoCookies() {
-		ChatHelper::info( __METHOD__ . ': Method called' );
+	public static function getSessionKey() {
+		self::info( __METHOD__ . ': Method called' );
 		$wg = F::app()->wg;
 
 		if ( !$wg->User->isLoggedIn() ) {
-			return [ "key" => false ];
+			return '';
 		}
-		$key = "Chat::cookies::" . sha1( $wg->User->getId() . "_" . microtime() . '_' .  mt_rand() );
-		$wg->Memc->set( $key, [ "user_id" => $wg->User->getId(), "cookie" => $_COOKIE ] , 60 * 60 * 48 );
+		$key = 'Chat::cookies::' . sha1( $wg->User->getId() . "_" . microtime() . '_' . mt_rand() );
+		$wg->Memc->set( $key, [
+			'user_id' => $wg->User->getId(),
```

Comment:
why not `wgMemc`, `wgUser` here? isn't `F::app()->wg` an old notation, a bit slower than mentioned?